CRM/Aoservicelisting/Form/ProviderApplicationConfirm.php

```diff
@@ -339,9 +342,11 @@ public function submit($values)
         E::createRelationship($abaMember['id'], $organization['id'], EMPLOYER_CONTACT_REL);
 
         // create address
-        $addressKey = $key - 1;
```

Comment:
@seamuslee001 could you remind me why this was being done?

Reply:
My only assumption is that the reason was we were starting with like row 2 assuming that the primary contact = row 1 but that has changed now